chsdi/templates/htmlpopup/bohrungen_tiefer_500.mako

```diff
@@ -0,0 +1,48 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+<%
+    lang = lang if lang in ('fr','it','en', 'rm') else 'de'
+    auftraggeb = 'auftraggeb_%s' % lang
+    rechtein = 'rechtein_%s' % lang
+    inhalt = 'inhalt_%s' % lang
+    auskunft = 'auskunft_%s' % lang
+
+%>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-bohrungen_tiefer_500.name')}</td>       <td>${c['attributes']['name'] or '-'}</td></tr>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-bohrungen_tiefer_500.auftraggeb_de')}</td>       <td>${c['attributes'][auftraggeb] or '-'}</td></tr>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-bohrungen_tiefer_500.rechtein')}</td>       <td>${c['attributes'][rechtein] or '-'}</td></tr>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-bohrungen_tiefer_500.inhalt')}</td>       <td>${c['attributes'][inhalt] or '-'}</td></tr>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-bohrungen_tiefer_500.download')}</td>       <td><a href="${c['attributes']['download'] or '-'}" target="_blank">Link</a></td></tr>
```

Comment:
Thx for the feedback! Indeed, I mispelled it... The misspelled images are now removed and I modified the .mako-file. I hope it's fine now.